src/ekore/anomalous_dimensions/unpolarized/time_like/as2.py

```diff
@@ -0,0 +1,693 @@
+
+    """
+    qqs1 = (
+        constants.CF
+        * nf
+        * (
+            (
+                4
+                * (
+                    8
+                    + 44 * N
+                    + 46 * npp(N, 2)
+                    + 21 * npp(N, 3)
+                    + 14 * npp(N, 4)
+                    + 15 * npp(N, 5)
+                    + 10 * npp(N, 6)
+                    + 2 * npp(N, 7)
+                )
+            )
+            / ((-1 + N) * npp(N, 3) * npp(1 + N, 3) * npp(2 + N, 2))
+        )
+    )
+    return qqs1
+
+
+@nb.njit(cache=True)
+def gamma_qg(N, nf):
+    r"""Compute the NLO quark-gluon anomalous dimension.
+
+    Implements :eqref:`B.10` from :cite:`Mitov:2006wy`
+    and :eqref:`A1` from :cite:`Gluck:1992zx`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment
+    nf : int
+        No. of active flavors
+
+    Returns
+    -------
+    gamma_qg : complex
+        NLO quark-gluon anomalous dimension
+        :math:`\gamma_{qg}^{(1)}(N)`
+
+    """
+    s1 = w1.S1(N)
+    s2 = w2.S2(N)
+    sm2 = w2.Sm2(N, s2, True)
+
+    qg1 = (
+        nf
+        * constants.CF
+        * (
+            (
+                2
+                * (
+                    8
+                    + 12 * N
+                    + 18 * npp(N, 2)
+                    + 77 * npp(N, 3)
+                    + 127 * npp(N, 4)
+                    + 104 * npp(N, 5)
+                    + 45 * npp(N, 6)
+                    + 9 * npp(N, 7)
+                )
+            )
+            / (npp(N, 3) * npp(1 + N, 3) * npp(2 + N, 2))
+            - (
+                4
+                * s1
+                * (
+                    -8
+                    - 12 * N
+                    + 22 * npp(N, 2)
+                    + 25 * npp(N, 3)
+                    + 10 * npp(N, 4)
+                    + 3 * npp(N, 5)
+                )
+            )
+            / (npp(N, 2) * npp(1 + N, 2) * npp(2 + N, 2))
+            + (4 * npp(s1, 2) * (2 + N + npp(N, 2))) / (N * (1 + N) * (2 + N))
+            - (20 * s2 * (2 + N + npp(N, 2))) / (N * (1 + N) * (2 + N))
+        )
+    )
+    qg2 = (
+        nf
+        * constants.CA
+        * (
+            (
+                4
+                * (
+                    144
+                    + 600 * N
+                    + 980 * npp(N, 2)
+                    + 2366 * npp(N, 3)
+                    + 2564 * npp(N, 4)
+                    + 379 * npp(N, 5)
+                    - 1177 * npp(N, 6)
+                    - 1037 * npp(N, 7)
+                    - 423 * npp(N, 8)
+                    - 76 * npp(N, 9)
+                    - 288 * zeta2 * npp(N, 2)
+                    - 720 * zeta2 * npp(N, 3)
+                    - 504 * zeta2 * npp(N, 4)
+                    + 180 * zeta2 * npp(N, 5)
+                    + 576 * zeta2 * npp(N, 6)
+                    + 504 * zeta2 * npp(N, 7)
+                    + 216 * zeta2 * npp(N, 8)
+                    + 36 * zeta2 * npp(N, 9)
```

Comment:
Collect as much as possible: it simplifies the evaluation, and improves readability.

Reply:
I had used mathematica simplifications to obtain all the expressions.